ctf/tests/core_suite.lua

```diff
@@ -0,0 +1,91 @@
+local lunatest = require("lunatest")
```

Comment:
Oh, I misread, the test folder is already in a mod. I forgot this wasn't a subgame. The structure is fine then